web/sites/default/config/core.extension.yml

```diff
@@ -14,6 +14,7 @@ module:
   breakpoint: 0
   bulk_user_registration: 0
   ckeditor: 0
+  clamav: 0
```

Comment:
This should not have been enabled locally I don't think. 🤔

Reply:
Fixed